File: CILantroTools/CILantroToolsWebAPI/CILantroToolsWebAPI/Search/SearchParameter.cs
namespace CILantroToolsWebAPI.Search
{
    public class SearchParameter
    {
        public SearchOrderParameter OrderBy { get; set; }

        public SearchOrderParameter OrderBy2 { get; set; }

        public SearchOrderParameter OrderBy3 { get; set; }

        public int PageSize { get; set; }

        public int PageNumber { get; set; }
    }
}

File: CILantroTools/cilantro-tools-web-ui/src/api/search/SearchParameter.ts
import SearchFilter from './SearchFilter';
import SearchOrderParameter from './SearchOrderParameter';

interface SearchParameter<TReadModel> {
	orderBy: SearchOrderParameter<TReadModel>;
	orderBy2?: SearchOrderParameter<TReadModel>;
	orderBy3?: SearchOrderParameter<TReadModel>;
	pageSize: number;
	pageNumber: number;
	filters: Array<SearchFilter<TReadModel>>;
}

export default SearchParameter;
